New post about GHelp statistics (#843)
usegalaxy-eu/website: _posts/2021-11-25-ghelp-stats.md

---

> site: [freiburg,elixir-it]

Reviewer: ```suggestion
site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
```

title: 'An overview on the Galaxy Help Forum'

> tags: [galaxy,support,data,society,report]

Reviewer: ```suggestion
tags: [galaxy, support, society, report]
```

supporters:
- galaxy-europe
---

> As part of the Galaxy Community, you probably have visited the [Galaxy Help Forum](https://help.galaxyproject.org/) (GHelp). This website, together with the different Gitter channels, is one of the primary meeting places where Galaxy users turn to in search of aswers. From the GHelp supporting group, we thank you for the trust you place in us every day.

Reviewer: ```suggestion
As part of the Galaxy Community, you have probably visited the [Galaxy Help Forum](https://help.galaxyproject.org/) (GHelp). This website, together with the different Gitter channels, is one of the primary meeting places where Galaxy users turn to searching for answers. From the GHelp supporting group, we thank you for the trust you place in us every day.
```

> Since November 2018, when it was put into service, more than 3.000 users have been registrated, and around 9.3000 posts have been written. Currently, the average number of daily visits is around 800 (fig. 1), having reached its peak in May 2021, with 2.800 visits in a single day.

Reviewer: ```suggestion
Since November 2018, when it was put into service, more than 3,000 users have been registered, and around 9.3000 posts have been written. Currently, the average number of daily visits is around 800 (fig. 1), having reached its peak in May 2021, with 2,800 visits in a single day.
```

Reviewer: 9.3000 is 93K?

Author: 9.300 sorry!

Reviewer: don't forget "registered" and the comma for the number (9,300) 😄

> Since November 2018, when it was put into service, more than 3.000 users have been registrated, and around 9.300 posts have been written. Currently, the average number of daily visits is around 800 (fig. 1), having reached its peak in May 2021, with 2,800 visits in a single day.

Reviewer: ```suggestion
Since November 2018, when it was put into service, more than 3,000 users have been registered, and around 9,300 posts have been written. Currently, the average number of daily visits is around 800 (fig. 1), having reached its peak in May 2021, with 2,800 visits in a single day.
```

![Visits last months](/assets/media/ghelp_stats/visits_two_months.png)
    
*Figure 1. Daily number of visits in the period between October and November 2021.*

The analysis of the records since 2019 shows that the average number of visits per year has risen from 464 to 822 visits in 2021 (fig. 2). A remarkable aspect of the trend in the increase in visits is the sharp spike that occurred in April 2020, coinciding with the origin of the pandemic (fig. 3). During this period, the average number of visits increased from around 600 to more than 800 visits per month. One of the factors that undoubtedly determined this increase was the growing interest of researchers in bioinformatics tools linked to COVID-19 research.

![Visits boxplot](/assets/media/ghelp_stats/visits_box.png)
    
*Figure 2. Description of the number of visits from 2019 to 2021.*
    

> Since then, the monthly number of visits has been around 800 users, peaking in May 2020. With respect to the total number of visits, between the period from January 1st to November 23st, the values recorded in 2019, 2020 and 2021 were 152.002, 264.426 and 268.793 respectively.

Reviewer: ```suggestion
Since then, the monthly number of visits has been around 800 users, peaking in May 2020. With respect to the total number of visits, between the period from January 1st to November 23st, the values recorded in 2019, 2020 and 2021 were 152,002, 264,426 and 268,793 respectively.
```

![Visits median](/assets/media/ghelp_stats/visits_median.png)
    
*Figure 3. Total number of visits from 2019 to 2021.*

> Regarding the number of posts, the average number of new publications per month since 2019 is above 200, peaking in May 2019, when 370 new entries were published (fig. 4). In the year 2021, from January 1th to November 23th, 2.895 new entries have been published.

Reviewer: ```suggestion
Regarding the number of posts, the average number of new publications per month since 2019 is above 200, peaking in May 2019, when 370 new entries were published (fig. 4). In 2021, from January 1st to November 23rd, 2,895 new entries have been published.
```

    
![Posts boxplot](/assets/media/ghelp_stats/posts_box.png)
    
*Figure 4. Description of the number of posts from 2019 to 2021.*

> To conclude this post, we would like to encourage all members of the Galaxy Community to participate in [GHelp](https://help.galaxyproject.org/). It is unquestionable that Science is a collective enterprise; the complexity of Nature makes it unmanageable for a single person or generation, since the diversity of Knowledge transcends any defined limit. 

Reviewer 2: That is a bit too transcendental for my taste ;)

> > The mutual-aid tendency in man has so remote an origin, and is so deeply interwoven with all the past evolution of the human race, that is has been maintained by mankind up to the present time, notwithstanding all vicissitudes of history. ― Peter Kropotkin

Reviewer: 😁
